versions.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 4.30.0"
+      version = ">= 4.40"
```

Comment:
why not keep it like this? And why 4.40 and not 4.40.0 if we really need to upgrade?